### ISSUE TITLE
add a panic / all off button

### ISSUE THREAD

**Issue by MonsieurPaulLeBoulanger** [monocasual/giada#148]
Hello,

I use giada to play samples & drums with my band, using a midi pedal board.

During rehearsal, I often need to stop all sounds from giada, I do it assigning a midi note to stop each sound. 
Would it be bossible to add a "panic" button to stop all sounds ?
That button should be keyboard or midi driven.

**Comment by monocasual** [comment]
Hi there, at the moment we have a panic-like effect with the "stop all channels" option in the Configuration window, "Behaviors" tab ([documentation](https://giadamusic.com/documentation/show/configuration)).  This is not a true panic mode however: FX tails (delays, reverbs, ....) won't get truncated. Is it enough for your scenario?

**Comment by MonsieurPaulLeBoulanger** [comment]
Hi,
This option works when using the sequencer and I don't  : I basically use giada as a sample player : every sound is set to "Oneshot Press", one sample can last few minutes.

So for now I've set the same midi note to the  "key release"  on all my samples, it does the job, but in my case, a "stop all" button / trigger would be awesome :)

**Comment by monocasual** [comment]
You are right, the "stop all channels" option works only when using the sequencer. So the panic button seems a sensible feature to add :) Thanks!

**Comment by MonsieurPaulLeBoulanger** [comment]
It's great if you can add this feature, thank you :)

**Comment by gvnnz** [comment]
Hey @blairmichaelg, sure go ahead! Please open a pull request when done. Thanks!
